src/openvic-simulation/country/CountryInstance.hpp

```diff
@@ -98,6 +101,8 @@ namespace OpenVic {
 
 		/* Budget */
 		fixed_point_t PROPERTY(cash_stockpile);
+		IndexedMap<Strata, slider_value_t> PROPERTY(tax_rate_by_strata);
```

Comment:
```suggestion
		IndexedMap<Strata, tax_slider_value_t> PROPERTY(tax_rate_by_strata);
```
If you add the type alias `using tax_slider_value_t = slider_value_t<0, 100>;` above (e.g. under `using unit_variant_t = uint8_t;`) then we can use it throughout this class rather than needing to explicitly give the template parameters each time (even if the template has default parameters we'd still have to write `slider_value_t<>` each time).